Jobsway2go.Core/Models/Collection.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Jobsway2go.Core.Models
+{
+    public class Collection
+    {
+        public int Id { get; set; }
+        public List<ApplicationUser> UserId { get; set; }
+        public string Name { get; set; }
```

Comment:
UserId is not a list. Research about entity framework virtual keyword for relationships in models.